quic/s2n-quic-platform/src/io/tokio.rs

```diff
@@ -90,6 +73,7 @@ impl Io {
             },
         });
 
+        // try to use the tokio runtime handle if provided
```

Comment:
can you also add a comment on the else case. is that case used when s2n-quic runs inside another program which already started the tokio runtime.

Reply:
yep